test/test_linalg.py

```diff
@@ -3198,7 +3199,8 @@ def test_cholesky_solve_out_errors_and_warnings(self, device, dtype):
     @precisionOverride({torch.float32: 2e-3, torch.complex64: 2e-3,
                         torch.float64: 1e-8, torch.complex128: 1e-8})
     def test_inverse(self, device, dtype):
-        from torch.testing._internal.common_utils import random_fullrank_matrix_distinct_singular_value
+        make_fullrank = make_fullrank_matrices_with_distinct_singular_values
+        make_arg = partial(make_fullrank, device=device, dtype=dtype)
```

Comment:
Do we really care about distinct singular values here?

Reply:
Nevermind, the condition number it is we care about.

Reply:
I'm planning to change the name to `make_wellconditioned` or something along those lines. It'll have the property that the matrix also has distinct singular values, and we'll document that in the function. In practice, it'll just be the function we'll use to generate samples for matrices in `linalg` for most tests